_pages/home.md

```diff
@@ -11,13 +11,24 @@ excerpt: "<br/><br/>"
 
 
 <div class="notice--info" markdown="1">
-  <h2 id="eacl-2023-call-for-papers">EACL 2023 templates</h2>
+  <h2 id="eacl-2023-abstract">Abstract submissions open</h2>
+  **October 4, 2022**. Abstract submissions are now enabled on Softconf at [this link](https://softconf.com/eacl2023/papers).
```

Comment:
We should also include the link in the call for papers here: "Submission is electronic, using the Softconf START conference management system."

Reply:
Done